### PR TITLE
migration of old urls to new site

### PR DESCRIPTION
**Pull request by tompainetoday**
this should work, depending on your server configurations. feel free to use or improve for your purposes. but please, make this migration happen. you have hundreds of links to the old url format that will no longer work without a proper redirect.

### CODE REVIEW

**Comment by dbactual** [comment]
This is useful, thanks for putting it together. 

I'm not too worried about engines re-indexing the site. There have been plenty of changes in the past, as well as new additions, which all get re-indexed. That being said, redirects would certainly help.

Do you think it might make more sense to reduce this to a few wildcard redirects, instead of a verbose listing of every file? It seems like most of this would boil down to things like: 

major-works -> works/major-works
essays -> works/essays
etc.

**Comment by tompainetoday** [comment]
> Do you think it might make more sense to reduce this to a few wildcard redirects, instead of a verbose listing of every file? It seems like most of this would boil down to things like:
> 
> major-works -> works/major-works essays -> works/essays etc.

Yes, absolutely! Save time and easier for all the static pages that now have new urls. I didn't do that since I figured you might add more writings in the future, and don't need those urls to redirect. But whatever you all feel is best! 

It's not so much about the reindexing, but the incoming links now coming into the site that go to a dead 404 page.

File: content/.htaccess [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+ Redirect 301 /questionable-authorship/useful-and-entertaining-hints.html /works/questionable-authorship/useful-and-entertaining-hints.html
+ Redirect 301 /recently-discovered/a-fifth-forester-letter.html /works/recently-discovered/a-fifth-forester-letter.html
+ Redirect 301 /recently-discovered/attack-on-paper-money-laws.html /works/recently-discovered/attack-on-paper-money-laws.html
+ Redirect 301 /recently-discovered/four-letters-on-interesting-subjects.html /works/recently-discovered/four-letters-on-interesting-subjects.html
+ Redirect 301 /recently-discovered/nonsense-from-new-york.html /works/recently-discovered/nonsense-from-new-york.html
+ Redirect 301 /recently-discovered/on-the-affairs-of-the-state.html /works/recently-discovered/on-the-affairs-of-the-state.html
+ Redirect 301 /recently-discovered/response-to-observations-on-the-american-revolution.html /works/recently-discovered/response-to-observations-on-the-american-revolution.html
+ Redirect 301 /recently-discovered/response-to-an-accusation-of-bribery.html /works/recently-discovered/response-to-an-accusation-of-bribery.html
```
Comment (dbactual) [inline content/.htaccess]:
```suggestion
RewriteEngine On

RewriteRule ^major-works(/.*)?$ /works/major-works$1 [L,NC,NE,R=301]
RewriteRule ^essays(/.*)?$ /works/essays$1 [L,NC,NE,R=301]
RewriteRule ^letters(/.*)?$ /works/letters$1 [L,NC,NE,R=301]
RewriteRule ^questionable-authorship(/.*)?$ /works/questionable-authorship$1 [L,NC,NE,R=301]
RewriteRule ^recently-discovered(/.*)?$ /works/recently-discovered$1 [L,NC,NE,R=301]
RewriteRule ^aboutpaine(/.*)?$ /pages/resources$1 [L,NC,NE,R=301]

RewriteRule ^membership.html /pages/membership.html
RewriteRule ^pages/the-official-collected-works-of-thomas-paine-project.html /pages/collected-works-project.html

 ```

Comment (dbactual) [reply content/.htaccess]: I think the above suggested change should handle everything we need for redirects. If it looks good to you, go ahead and commit the changes into this branch and then we can merge to update the live site.

**Review by dbactual — APPROVED** [review]

**Pull request merged**